common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -287,21 +287,45 @@ status_function() {
     pgrep -l ieee1905_transport
     pgrep -l local_bus
 
-    # check for operational status
-    LOGS_PATH=@BEEROCKS_LOG_FILES_PATH@
+    bridge_mac="$(ip link show dev @BEEROCKS_BRIDGE_IFACE@ | awk '/^ *link\/ether / {print $2}')"
+    bml_cmd="bml_get_device_operational_radios $bridge_mac"
+
+    if  pgrep -l beerocks | grep -q beerocks_cont ; then
```

Comment:
Can't we just do
```suggestion
    if  pgrep -l beerocks_controller
```

Reply:
For some reason the pgrep output is cut short and cannot find "beerocks_controller"
So I changed it to use
 ` if  pgrep -l beerocks_cont ; then' instead